.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+app/wasm/**/* @nicolaslara @czarcas7ic
```

Comment:
`/**/*` is redundant.

`app/wasm/` should be enough

Reply:
Same for others: `x/twap/`

Reply:
AFAIK, if we do not use /**/*, it would only match the directory itself, but not any of its subdirectories within the directory. 

Can you double confirm if this is the case? It might be something I have wrong knowledge of, if so, would make the according changes

Reply:
```
# In this example, @doctocat owns any files in the build/logs
# directory at the root of the repository and any of its
# subdirectories.
/build/logs/ @doctocat
```

from: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

Reply:
thanks, removed and revised!